src/integration/java/io/pinecone/integration/dataPlane/UpsertAndQueryPodTest.java

```diff
@@ -185,22 +185,16 @@ public void upsertOptionalVectorsAndQueryIndexFutureTest() throws InterruptedExc
 
     @Test
     public void upsertNullSparseIndicesNotNullSparseValuesFutureTest() {
-        AsyncIndex dataPlaneClient = new AsyncIndex(connection);
-        StringBuilder exceptionMessage = new StringBuilder();
         String id = RandomStringBuilder.build(3);
         try {
-            dataPlaneClient.upsert(id,
+            asyncIndexClient.upsert(id,
                     generateVectorValuesByDimension(dimension),
                     null,
                     generateVectorValuesByDimension(dimension),
                     null,
                     null).get();
-        } catch (PineconeValidationException validationException) {
```

Comment:
I don't think this was needed, I think we can just use one `catch` statement.

Reply:
It is a bit cleaner if we just declare the exceptions in the test method signature that we don't want to handle (i.e. we want them to cause a failure if they're thrown)

Example:
```java
public void upsertNullSparseIndicesNotNullSparseValuesFutureTest() throws ExecutionException, InterruptedException {
  /// ...
  try {
     // ...

    fail("should have thrown PVE")

  } catch (PVE expected) {
    // validate exception message
  }
```

Reply:
Ah thank you! I wasn't very clear on how to properly represent this.